source/exploring_catalog_and_datasets/04_getting_involved/exporting_data.rst

```diff
@@ -4,44 +4,68 @@ Exporting data
+1. Go to the **Export** tab of the dataset.
+2. Choose the file format to download.
+3. Perform one of the following actions:
 
-- CSV
-- JSON
-- Excel
-- GeoJSON
-- Shapefile
-- KML
+   * Click **Whole dataset** next to the desired format.
+   * If the dataset is filtered and you want to export the filtered data only, click **Only the selected records** next to the desired format.
 
 .. admonition:: Note
    :class: note
 
-   It is possible to download the dataset in other formats using the `Opendatasoft Search API <https://help.opendatasoft.com/apis/ods-search-v2/#exporting-records>`_.
+   - You can use the parameter :code:`use_labels_for_header=true` in the exports links to use the labels instead of the technical identifiers of the fields as a header (not available in the Opendatasoft Search API v2).
+   - You can also export datasets in other formats using the `Opendatasoft Search API v2 <https://help.opendatasoft.com/apis/ods-search-v2/#exporting-records>`_.  
 
-.. admonition:: Note
-   :class: note
+Available export formats
+------------------------
 
-   There are 2 limitations in Shapefile exports.
+In the Export tab of a dataset, the following formats can be available:
 
-   - The export cannot contain more than 50,000 records.
-   - The export cannot contain more than one geographical shape type (e.g. point, polygon, linestring etc.). Only the first type of the first non-empty record will be kept in the export.
+Flat file formats
+~~~~~~~~~~~~~~~~~
 
-1. Go to the Export tab of the dataset.
-2. Choose the file format to download.
-3. Click on "Whole dataset" next to the chosen format.
+.. list-table::
+   :header-rows: 1
+   :widths: 25 75
+
+   * * Format
+     * Notes
+   * * CSV
+     * CSV uses semicolon (;) as a separator. 
+   * * JSON
+     * 
+   * * Excel
+     * 
+
+Geographic file formats
+~~~~~~~~~~~~~~~~~~~~~~~
 
 .. admonition:: Important
-   :class: important
+   :class: Important
 
-   If the dataset is filtered, it is possible to only download the filtered data. To do so, click on "Only the selected records" instead of "Whole dataset".
+   The geographical coordinates of a dataset are expressed in WGS84 by default. When available, you can select additional geographical projections from the drop-down displayed above the export formats.
 
-.. admonition:: Note
-   :class: note
+.. list-table::
+   :header-rows: 1
+   :widths: 25 75
+
+   * * Format
+     * Notes
+   * * GeoJSON
+     * 
+   * * Shapefile
+     * There are 2 limitations in Shapefile exports:
 
-   You can use the parameter :code:`use_labels_for_header=true` in the exports links to use the labels instead of the technical identifiers of the fields as a header (not available in API v2).
+        - The export cannot contain more than 50,000 records.
+        - The export cannot contain more than one geographical shape type (point, polygon, linestring, etc.). Only the first type of the first non-empty record will be kept in the export.
```

Comment:
When I read this phrase, I think of two different cases:
- a dataset with a "geoshape" field, in this field different types (point, polygon…), only the shapes of the same kind that the first shape found are kept in the export (but what about the data attached to the other shapes? are the records kept without their shape? or not present at all?)
- a dataset with multiple "geo" field (geoshape or geo_point_2d): I tried with https://public.opendatasoft.com/explore/dataset/georef-united-states-of-america-place-millesime/information/ : in the Information > Data model tab, there are both fields available, and geo_point_2d comes first; when exporting as Shapefile and looking at the export with QGIS, only the shape is present

I suppose that you were thinking about the first case, but maybe that the second one should be explained too?

Reply:
Fixed in 9da1777ab424022b02d96e9ff6512efdd7579538